### PR TITLE
TELCODOCS-1083: Added x86_64 and arm64 as supported CPU architectures.

### PR DESCRIPTION
**Pull request by kdrake20**
Fixes: [TELCODOCS-1083](https://issues.redhat.com/browse/TELCODOCS-1083)

For: Version 4.13 

Doc Preview [here](https://56752--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_sno/install-sno-preparing-to-install-sno.html#install-sno-requirements-for-installing-on-a-single-node_install-sno-preparing)

Signed-off-by: Katie Tothill ktothill@redhat.com

### CODE REVIEW

**Comment by ktothill** [comment]
@chadcrum @oourfali, could I please get a review of this PR? Thank you!

**Comment by ktothill** [comment]
**Who:** 
Chad Crum, QE
Oved Ourfali, SME

**Summary:** Requested QE, SME and Peer Reviews.

**Next Steps:** Receive `LGTM` from QE and Peer Review.

**Comment by chadcrum** [comment]
Looks right to me

@trewest

File: modules/install-sno-requirements-for-installing-on-a-single-node.adoc [hunk 1]
```diff
@@ -9,7 +9,9 @@ Installing {product-title} on a single node alleviates some of the requirements
 
 * *Administration host:* You must have a computer to prepare the ISO, to create the USB boot drive, and to monitor the installation.
 
-* *Supported platforms:* Installing {product-title} on a single node is supported on bare metal and link:https://access.redhat.com/articles/973163[Certified third-party hypervisors]. In all cases, you must specify the `platform.none: {}` parameter in the `install-config.yaml` configuration file.
+* *CPU Architecture:* Installing {product-title} on a single node supports `x86-64` and `arm64` CPU architectures.
```
Comment (trewest) [inline modules/install-sno-requirements-for-installing-on-a-single-node.adoc:12]: nit: x86_64

**Pull request merged**